install/01_enabling_pg_stat_statements.mdx

```diff
@@ -32,24 +32,67 @@ sudo yum install postgresql-contrib
 
 ## Configure PostgreSQL
 
-Add the following entries to your postgres.conf:
+There are a number of settings that define how pg_stat_statements will collect
+statistics.
+The right configuration will depend on your server, workload, and preferences,
+but the following is the example of what to modify.
+You can apply these changes by updating your postgresql.conf file.
 
 ```
+# postgresql.conf
 shared_preload_libraries = 'pg_stat_statements'
 
-# Track statements generated by stored procedures as well
 pg_stat_statements.track = all
-
-# Increase the maximum number of statements tracked
 pg_stat_statements.max = 10000
-
-# Enable tracking of per-query I/O statistics
-track_io_timing = 1
+track_io_timing = on
 ```
 
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Value</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <a target="_blank" href="https://www.postgresql.org/docs/current/pgstatstatements.html#PGSTATSTATEMENTS-CONFIG-PARAMS">
+          <code>pg_stat_statements.track</code>
+        </a>
+      </td>
+      <td>all (default: top)</td>
+      <td>Enables tracking of queries inside stored procedures or functions.</td>
+    </tr>
+    <tr>
+      <td>
+        <a target="_blank" href="https://www.postgresql.org/docs/current/pgstatstatements.html#PGSTATSTATEMENTS-CONFIG-PARAMS">
+          <code>pg_stat_statements.max</code>
+        </a>
+      </td>
+      <td>10000 (default: 5000)</td>
+      <td>
+        Increases the maximum number of statements tracked.
+        Recommended to increase further more if the database tends to have lots of distinct statements.
+        Restart is required for changes to take effect.
```

Comment:
```suggestion
        Increases the maximum number of statements tracked.
        Recommended to increase further if the database tends to have lots of distinct queries.
        Restart is required for changes to take effect.
```
I think "queries" instead of "statements" in one of these sentences helps tie this back to query activity, which is primarily the statements most users will be interested in.

I'm also tempted to add something like
> Queries that have not been seen recently will be pruned once this number is hit. The pruning happens in batches, so you may see the count less than the max even if the max has been reached

because we do get questions about that occasionally. That's probably too much text for the box. Maybe a footnote? Or we can leave it off if you think it doesn't fit anywhere.

Reply:
I think that "Queries that have not been seen recently will be pruned..." sentence makes sense to have it here. That sentence actually goes nicely with `pg_stat_statements_reset()` and `pg_stat_statements_info` view, so let me try to add some section to talk about it